<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zchxMainWindow</class>
 <widget class="QMainWindow" name="zchxMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	background-color: rgb(0,127,127);
	font-family:Microsoft Yahei;
	font-weight:normal;
	font-size:12pt;
	color:white;
}

QMenuBar{
	background:transparent;
}


QFrame{
	background-color: rgb(53,75,152);
}

QLabel{
	background-color: transparent;
}

QPushButton{
	border:1px solid white;	
	border-radius:4px;	
	background-color: transparent;
}

QPushButton::hover{
	background-color: green;
}

#ch1_btn,#ch2_btn{
    width:120px;
	height:60px;
	font-size:16pt;
	font-weight:bold;		
	background-color: rgb(105, 113, 113);
}

#ch1_btn:flat,#ch2_btn:flat{	
	background-color: rgb(0,250,0);
}

#ch1_btn::checked, #ch2_btn::checked{
	background-color: rgb(191,98,0);
}

QLineEdit{
	border:1px solid rgb(180, 180, 180);
	background: transparent;
	border-radius: 2px;
}

QLineEdit::hover{
  color:black;
  border-color:rgb(50,480,40);
  background-color:green;
}


  QMenuBar::item {
      spacing: 3px; /* spacing between menu bar items */
      padding: 1px 4px;
      background: transparent;
      border-radius: 4px;
  }

  QMenuBar::item:selected { /* when selected using mouse or keyboard */
      background: #a8a8a8;
  }

  QMenuBar::item:pressed {
      background: #888888;
  }

QMenu {
      background-color: rgb(0,127,127);
      margin: 2px; /* some spacing around the menu */
  }

  QMenu::item {
      padding: 2px 25px 2px 20px;
      border: 1px solid transparent; /* reserve space for selection border */
  }

  QMenu::item:selected {
      border-color: darkblue;
      background: rgba(100, 100, 100, 150);
  }

  QMenu::icon:checked { /* appearance of a 'checked' icon */
      background: gray;
      border: 1px inset gray;
      position: absolute;
      top: 1px;
      right: 1px;
      bottom: 1px;
      left: 1px;
  }

  QMenu::separator {
      height: 2px;
      background: lightblue;
      margin-left: 10px;
      margin-right: 5px;
  }

  QMenu::indicator {
      width: 13px;
      height: 13px;
  }

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="btn_frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="ch1_btn">
         <property name="text">
          <string>通道一</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ch2_btn">
         <property name="text">
          <string>通道二</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="zchxRadarWidget" name="Ch1" native="true"/>
    </item>
    <item>
     <widget class="zchxRadarWidget" name="Ch2" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>开机</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>zchxRadarWidget</class>
   <extends>QWidget</extends>
   <header>zchxradarwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
